src/plugins/subscription-manager.py

```diff
@@ -159,6 +159,17 @@ def warnOrGiveUsageMessage(conduit):
             conduit.info(2, msg)
 
 
+def init_hook(conduit):
+    disable_system_repos = conduit.confBool('main', 'disable_system_repos', default=False)
+
+    if disable_system_repos:
+        repo_storage = conduit.getRepos()
+        for repo in repo_storage.repos.values():
+            if os.path.basename(repo.repofile) != "redhat.repo":
+                conduit.info(3, 'Disable repository %s of %s' % (repo.id, repo.repofile))
+                repo_storage.disableRepo(repo.id)
+
```

Comment:
When at least one system repository was disabled, then I would be nice to display some explanation why system repositories were disabled. Something like this `"subscription-manager plugin disabled %d system repositories with respect of configuration in /etc/yum/pluginconf.d/subscription-manager.conf"`.